responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -104,6 +104,32 @@ def __init__(self, model: Optional[Any], train: pd.DataFrame,
         :type feature_metadata: FeatureMetadata
         """
         categorical_features = categorical_features or []
+        if len(test) > maximum_rows_for_test:
```

Comment:
Should this change be merged to postGA branch?

Reply:
Thanks. I updated the target branch.